scripts/extensions/datetimeField/src/getTemplateDateTimeField.tsx

```diff
@@ -0,0 +1,230 @@
+import {ISuperdesk, ITemplateEditorComponentProps} from 'superdesk-api';
+import * as React from 'react';
+import set from 'date-fns/set';
+import format from 'date-fns/format';
+import addMinutes from 'date-fns/addMinutes';
+import {IDateTimeFieldConfig} from './extension';
+import {DatePickerISO, TimePicker, Button, Switch} from 'superdesk-ui-framework/react';
+import {Radio, CheckGroup} from 'superdesk-ui-framework';
+
+interface ITemplateDateTimeFieldState {
```

Comment:
It's a local interface that isn't exported anywhere, you can name it simply `IState`.